example/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: untitled6
```

Comment:
untilted6? I think it would be better to roll with standard `mq_theme_tools_gen_example`

Reply:
Changed as suggested, thx